environment.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   - numpy
   - networkx
   - rdkit
+  - packaging
```

Comment:
I think this is the healthiest way of doing (at least MDA just puts a direct dependency on packaging)

Reply:
Sounds good!